deploy/k8s_deployments/setup_flex.sh

```diff
@@ -64,13 +64,11 @@ else
 fi
 
 
-# Run a tail -f on the flex log file (which locate on the host), so it will be visible by running kubectl logs <flex POD>
-tail -F /var/tmp/ubiquity-k8s-flex.log &
+# Run a tail -F on the flex log file (which locate on the host), so it will be visible by running kubectl logs <flex POD>
+tail -F /usr/libexec/kubernetes/kubelet-plugins/volume/exec/ibm~ubiquity-k8s-flex/ubiquity-k8s-flex.log &
```

Comment:
sorry
inside the flex container the "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/ibm~ubiquity-k8s-flex/" is actually mapped to /mnt/flex/ibm~ubiquity-k8s-flex

So please replace the  "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/ibm~ubiquity-k8s-flex/"  with an existing variable that already container the  /mnt/flex/ibm~ubiquity-k8s-flex. So just use ${MNT_FLEX_DRIVER_DIR}

instead of 
tail -F /usr/libexec/kubernetes/kubelet-plugins/volume/exec/ibm~ubiquity-k8s-flex/ubiquity-k8s-flex.log &
Use
tail -F  ${MNT_FLEX_DRIVER_DIR}/ubiquity-k8s-flex.log &